src/ensembl/README.md

```diff
@@ -1,38 +1,46 @@
-# Ensembl 2020 Frontend
+# Running the ensembl-client application
 
-## Setup
+## Installation
 
-The tools that need setting up are Node.js and the IDE. To setup Node.js you only need to run `npm install`. This will install all the necessary NPM packages for app development and the build.
+The Ensembl client runs on NodeJS, and all supplementary libraries required for the genome browser (e.g. Rust) are included in this repository. To setup Node.js you only need to run `npm install`. This will install all the necessary NPM packages for app development and the build.
```

Comment:
> NodeJS

Node.js is the correct spelling:

![image](https://user-images.githubusercontent.com/6834224/75975058-c1bf4380-5ecf-11ea-9aa5-624ddfdac16c.png)
([link](https://nodejs.org/en/))

Reply:
> To setup Node.js you only need to run `npm install`.

This is incorrect 😄 To setup Node, you need to install Node on your computer (we can provide a list of steps for doing that or add a link to instructions for how to do it). `npm install` installs the necessary dependencies for our project.

Reply:
I was just rewording the existing instructions.

Reply:
Yes. We now have a chance to improve them 😄

Reply:
OK, moved installation instructions to the main README, so the ensembl/src README is just the detailed npm commands.